docs/connectors/building-connectors/data-types.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Data types"
```

Comment:
I'm thinking if this is the right place for this? Sure you care about raw vs structured data when you're developing connectors, but also when you're using conduit. WDYT about moving this to the [OpenCDC record](https://conduit.io/docs/features/opencdc-record) page?

Reply:
Sounds reasonable to me. :+1: